public/static/js/spawned_hosts.js

```diff
@@ -272,65 +272,66 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', 'mciS
       );
     };
 
-    $scope.updateHostExpiration = function () {
-      let new_expiration = null;
-      if (!$scope.curHostData.no_expiration) {
-        new_expiration = new Date($scope.curHostData.current_expiration);
-      }
-
-      mciSpawnRestService.extendHostExpiration(
-        'extendHostExpiration',
-        $scope.curHostData.id, new_expiration, {}, {
-          success: function (resp) {
-            window.location.href = "/spawn";
-          },
-          error: function (resp) {
-            notificationService.pushNotification('Error extending host expiration: ' + resp.data.error, 'errorHeader');
-          }
+    $scope.update = function () {
```

Comment:
Shouldn't updateRDPPassword from line 260 also be part of this?

Reply:
Nope because that's still a separate button. There were some layout issues with it that I fixed just now though

Reply:
Does it need to be a separate button?

Reply:
Yes because it opens a new popup, and you change the password there